src/AMOEBA/pair_amoeba.cpp

```diff
@@ -984,6 +1055,13 @@ void PairAmoeba::init_style()
 
   // request standard neighbor list
 
+
```

Comment:
More debug comments. The commented out code would not even work anymore if uncommented.